python/ql/lib/change-notes/2024-05-17-maD-rich-type-column.md

```diff
@@ -0,0 +1,4 @@
+---
+category: feature
+---
+* A Python MaD row may now contain a dotted path in the `type` column. Like in Ruby, a path to a class will refer to instances. This means that the summary `["foo", "Member[MyClass].Instance.Member[instance_method]", "Argument[0]", "ReturnValue", "value"]` can now be written `["foo.MS_Class", "Member[instance_method]", "Argument[0]", "ReturnValue", "value"]`. To refer to an actual class, one may add a `!` at the end of the path.
```

Comment:
Does this only work at the end of a path, or would `foo.bar!.baz` be meaningful?
(Edit: Looking at the code, it seems the answer is "no" for the time being.)

Reply:
Perhaps "MaD" should be spelled out in full, so as to make it a bit less jargony?

Reply:
MaD has been spelled out. Did you also want some reformulation?